torch/_C/_functions.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import AnyStr, List
+
+class UndefinedGrad:
+    def __init__(self) -> None: ...
+    def __call__(self, inputs: List) -> List: ...
```

Comment:
Nit: it would be better to explicitly say `List[Any]` (or really, `List[Tensor]` I think is more accurate here), as `List` will get errors from more strict mypy settings.

Reply:
Done!